core/state/state_object.go

```diff
@@ -205,40 +198,9 @@ func (c *StateObject) St() Storage {
 	return c.storage
 }
 
-//
-// Gas setters and getters
-//
-
 // Return the gas back to the origin. Used by the Virtual machine or Closures
 func (c *StateObject) ReturnGas(gas, price *big.Int) {}
```

Comment:
StateObject implements vm.ContextRef (it is the toplevel context).
We'll clean this up some other time.